apps/src/lib/script-editor/FlexGroup.jsx

```diff
@@ -112,7 +113,7 @@ const FlexGroup = React.createClass({
     }
     let l = `${this.normalizeLevelKind(level.kind)} '${key.replace(/'/, "\\'")}'`;
     if (active === false) {
-      l += ', active: false'
+      l += ', active: false';
```

Comment:
I just recently made a change to this file. how come the linter didnt complain at me when i did?